src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1488,7 +1488,7 @@ public function isMethodIdempotent()
      *
      * @see https://tools.ietf.org/html/rfc7231#section-4.2.3
      *
-     * @return bool True for GET and HEAD, false otherwise
```

Comment:
quite useful information to know which methods are considered "cacheable" by this method? (should probably be added to the normal description)

Reply:
The other similar methods don't have it. The implementation should know what cacheable, not the description.